uni/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -124,10 +125,11 @@ class AppSharedPreferences {
     await prefs.setString(locale, appLocale);
   }
 
-  static Future<Locale> getLocale() async {
+  static Future<AppLocale> getLocale() async {
     final prefs = await SharedPreferences.getInstance();
     final appLocale = prefs.getString(locale) ?? Platform.localeName;
-    return Locale(appLocale);
+
+    return appLocale == 'pt' ? AppLocale.pt : AppLocale.en;
```

Comment:
Store the model serialized to avoid checking against this hardcoded string: https://stackoverflow.com/questions/27673781/enum-from-string

Reply:
@DGoiana can you take a look at this?

Reply:
Changed in the last commit

Reply:
Hm, does `enum`'s `name` getter really prepend the class name to the variant name? I didn't have that idea from looking at the SO post

Reply:
`enum`'s `name` returns `'en'` or `'pt'`, depending on the locale